nano/node/bootstrap/bootstrap_lazy.hpp

```diff
@@ -0,0 +1,113 @@
+#pragma once
+
+#include <nano/node/bootstrap/bootstrap_attempt.hpp>
+#include <nano/node/bootstrap/bootstrap_bulk_pull.hpp>
+#include <nano/node/common.hpp>
+#include <nano/node/socket.hpp>
+#include <nano/secure/blockstore.hpp>
+#include <nano/secure/ledger.hpp>
+
+#include <boost/multi_index/hashed_index.hpp>
+#include <boost/multi_index/member.hpp>
+#include <boost/multi_index/ordered_index.hpp>
+#include <boost/multi_index_container.hpp>
+#include <boost/thread/thread.hpp>
+
+#include <atomic>
+#include <future>
+#include <queue>
+#include <unordered_set>
```

Comment:
Not sure all includes are needed: `<future>`, `<boost/thread/thread.hpp>` & `<nano/secure/ledger.hpp>`. Maybe others

Reply:
Updated